src/Orleans/Serialization/OrleansJsonSerializer.cs

```diff
@@ -7,16 +7,20 @@
 
 namespace Orleans.Serialization
 {
-    internal class OrleansJsonSerializer : IExternalSerializer
+    using Orleans.Providers;
+
+    public class OrleansJsonSerializer : IExternalSerializer
     {
+        public const string UseFullAssemblyNamesProperty = "UseFullAssemblyNames";
+        public const string IndentJsonProperty = "IndentJSON";
         private static JsonSerializerSettings defaultSettings;
         private Logger logger;
-
+        
         /// <summary>
         /// Returns a configured <see cref="JsonSerializerSettings"/> 
         /// </summary>
         /// <returns></returns>
-        internal static JsonSerializerSettings GetDefaultSerializerSettings()
+        public static JsonSerializerSettings GetDefaultSerializerSettings()
```

Comment:
This method was essentially moved from `SerializationManager`, but it's used by external providers in order to get serialization settings which are suitable for serializing types which we use (grain refs, etc).